routes/web.php

```diff
@@ -51,7 +51,9 @@
 
     if (OAuth::enabled()) {
         Route::get(config('statamic.oauth.routes.login'), [OAuthController::class, 'redirectToProvider'])->name('oauth.login');
-        Route::get(config('statamic.oauth.routes.callback'), [OAuthController::class, 'handleProviderCallback'])->name('oauth.callback');
+        Route::match(['get', 'post'], config('statamic.oauth.routes.callback'), [OAuthController::class, 'handleProviderCallback'])
+            ->withoutMiddleware('App\Http\Middleware\VerifyCsrfToken')
+            ->name('oauth.callback');
```

Comment:
Why is a POST request now required? Where is that explained?

Reply:
It's mentioned in the original bug report (https://github.com/statamic/cms/issues/9535) but [also mentioned here](https://socialiteproviders.com/Saml2/#installation-basic-usage).

When this route didn't accept `POST` requests, I got a 404 response.

Reply:
Thank you. 👍